### PR TITLE
Update chart dependencies.

### PR DESCRIPTION
**Pull request by mgruner**
#### What this PR does / why we need it

- Update some chart dependencies

#### Checklist

- [x] Chart Version bumped

### CODE REVIEW

File: zammad/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: zammad
-version: 12.2.2
```
Comment (monotek) [inline zammad/Chart.yaml:3]: if the dependencies change with major version i would also raise the charts major verion.

Comment (mgruner) [reply zammad/Chart.yaml:3]: @monotek thanks for the hint. I actually wanted to ask how we can perform a major update of the postgresql chart in future (as it requires manual work), but you already answered it. I went back to compatible updates for this change, don't want to create a major update now. Guess we'll need to collect some more breaking changes and upgrade all subcharts to the latest majors in a future release.

I also added the upgrade option for chart-testing, let's see if it works.

Could you please add my user `mgruner` as an admin to the `zammad` organisation on artifacthub?

Comment (monotek) [reply zammad/Chart.yaml:3]: invite for artiffacthub is out...

**Review by monotek — APPROVED** [review]

File: zammad/Chart.yaml (hunk 1 above)
Comment (mgruner) [reply zammad/Chart.yaml:3]: Thank you so much @monotek!

**Pull request merged**